src/Compilers/CSharp/Portable/Errors/ErrorCode.cs

```diff
@@ -2164,6 +2164,14 @@ internal enum ErrorCode
         ERR_BadSpecialByRefUsing = 9104,
         #endregion
 
+        #region diagnostics introduced for C# preview
```

Comment:
You can put all these error codes straight into C# 12 bucket above. All of C# 12 is preview until it is released.

Reply:
will do!